config/targets.conf

```diff
@@ -165,915 +165,918 @@ fe-som-rk3399             edge            hirsute     cli                      s
+# nanopi-r1 - degraded to CSC
+#nanopi-r1                 current         focal       minimal                  stable         yes
+#nanopi-r1                 current         buster      cli                      stable         yes
+#nanopi-r1                 current         bullseye    cli                      stable         yes
+#nanopi-r1                 current         focal       cli                      stable         yes
+#nanopi-r1                 edge            hirsute     cli                      stable         yes
 
 
-# nanopi-r2s 
-nanopi-r2s                current         focal       minimal                  stable         yes
-nanopi-r2s                current         buster      cli                      stable         yes
-nanopi-r2s                current         bullseye    cli                      stable         yes
-nanopi-r2s                current         focal       cli                      stable         yes
-nanopi-r2s                edge            hirsute     cli                      stable         yes
+# nanopi-r2s - degraded to CSC
+#nanopi-r2s                current         focal       minimal                  stable         yes
+#nanopi-r2s                current         buster      cli                      stable         yes
+#nanopi-r2s                current         bullseye    cli                      stable         yes
+#nanopi-r2s                current         focal       cli                      stable         yes
+#nanopi-r2s                edge            hirsute     cli                      stable         yes
 
-# nanopi-r2c 
-nanopi-r2c                current         focal       minimal                  stable         yes
-nanopi-r2c                current         buster      cli                      stable         yes
-nanopi-r2c                current         bullseye    cli                      stable         yes
-nanopi-r2c                current         focal       cli                      stable         yes
+# nanopi-r2c - CSC
+#nanopi-r2c                current         focal       minimal                  stable         yes
+#nanopi-r2c                current         buster      cli                      stable         yes
+#nanopi-r2c                current         bullseye    cli                      stable         yes
+#nanopi-r2c                current         focal       cli                      stable         yes
 
 # nanopi-r4s
 nanopi-r4s                current         focal       minimal                  stable         yes
-nanopi-r4s                current         buster      cli                      stable         yes
-nanopi-r4s                current         bullseye    cli                      stable         yes
+#nanopi-r4s                current         buster      cli                      stable         yes
+#nanopi-r4s                current         bullseye    cli                      stable         yes
```

Comment:
What's the reasoning behind dropping debian while keeping ubuntu? Is it the preference of the currently assigned maintainer? Can a maintainer choose to only support one or the other? Before I fill in the form and pick SBCs it would be useful to understand the process here.

Reply:
Reasoning was to lower workload by only having one flavour supported anymore. 
At the last release meeting (https://forum.armbian.com/topic/19104-armbian-2111-sambar-release-thread/?tab=comments#comment-129792) it was discussed, and as there did not seem to be a clear preference for debian/ubuntu, one of them (ubuntu) was just arbitrarily picked.

It would surely be possible and make sense to let the maintainer have a word on this.  Also note these changes are definitely not final - thats why we do the release meetings, every few months, to reevaluate changes and discuss further direction.

Reply:
Board maintainers are always free to add flavors as they like. However by default only one is shipped as mentioned.

Reply:
Would it make sense to put a poll on the forum for users if they would prefer supported cli images to be debian or ubuntu, given one of them?

As you say maintainers will of course have final word, but at least it can provide input in case there's a clear community preference for one over the other.